about/pricing.html.markerb

```diff
@@ -154,6 +154,28 @@ regionSelect.addEventListener("change", function () {
 
 For stopped Machines we charge only for the root file system (rootfs) needed for each Machine. Each 1GB of rootfs for a Machine stopped for 30 days is $0.15. The amount of rootfs needed is defined by your OCI image generated on your app plus a few [containerd](https://containerd.io/+external) tweaks on the underlying file system.
 
+### Machine Reservation Blocks
+
+We have a block-based commit scheme. Reserve a block of compute time, either for `performance` Machines or `shared` Machines, for a 40% discount. Reservations apply to any number of Machines in that class, in a specific region.
+
+The available reservation sizes are:
+
+**Performance Machines**
+
+* $144/yr for $20/mo of usage (i.e. $12/mo amortised with a 12 month commit)
```

Comment:
I think it might be better to explain this: `(i.e. $12/mo amortised with a 12 month commit)` in the description above rather than inline on only one price. Something like:

> We amortize the yearly cost over the year and you make a 12 month commitment. For example, if you buy a reservation block of $360/yr for $50/mo of usage, you'll pay $30/mo for 12 months.

Reply:
Ah we don't actually do that, you pay the $360 upfront. But I get the point, we should put this in proper prose.